javascript/classes.json

```diff
@@ -0,0 +1,244 @@
+{
+  "javascript": {
+    "classes": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Classes",
+        "support": {
+          "webview_android": {
+            "version_added": null
+          },
+          "chrome": [
+            {
+              "version_added": "49"
+            },
+            {
+              "version_added": "42",
+              "notes": "Requires strict mode. Non-strict mode support is behind the flag \"Enable Experimental JavaScript\", disabled by default."
+            }
+          ],
+          "chrome_android": {
+            "version_added": "56"
```

Comment:
Where does this 56 comes from?

Reply:
No idea, I'm putting `true` here and below.